scripts/windows/msys2_prepare.bat

```diff
@@ -13,13 +16,16 @@ rem Set required vars defaults
 IF "%ERTS_VERSION%"=="" SET "ERTS_VERSION=9.3"
 IF "%OTP_VERSION%"=="" SET "OTP_VERSION=20.3"
 IF "%WIN_OTP_PATH%"=="" SET "WIN_OTP_PATH=C:\Program Files\erl"
+IF "%WIN_JDK_BASEPATH%"=="" SET "WIN_JDK_BASEPATH=C:\Program Files\Java"
```

Comment:
Why is JDK needed? This is run by user - see docs/build-windows.md
If not needed by user, it looks like this shall be separate script for CI only?

Reply:
The files under `scripts/windows` are meant for users wanting to build the node. Therefore, the JDK is required to generate the swagger test client. So far there wasn't a differentiation between such a user and a developer using Windows.